### PR TITLE
Fix ObjectTypeReferenceTest, ObjectEquivalencyTest & ObjectWithPrivateFieldsNegativeTest  for project API jballerina-unit-tests migration

### PR DESCRIPTION
**Pull request by pramodya1994**
## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-unit-test/src/test/resources/testng.xml [hunk 1]
```diff
@@ -780,27 +776,7 @@
...
-                    <exclude name="testObjectEquivalenceWhenFieldsHavePublicVisibility"/>
-                    <exclude name="testObjectEquivalencyWithArguments"/>
-                    <exclude name="testObjectMemberOrder"/>
-                    <exclude name="testRuntimeEqPublicObjects"/>
-                    <exclude name="testRuntimeEqPublicObjects1"/>
-                    <exclude name="testSubtypingBetweenNonClientAndClientObject"/>
-                    <exclude name="testTupleMatchWithObjectEquivalency"/>
-                </methods>
-            </class>
+            <class name="org.ballerinalang.test.object.ObjectEquivalencyTest"/>
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/resources/testng.xml:779]: I think these are already included in https://github.com/ballerina-platform/ballerina-lang/pull/26797/files#diff-3835b6074874bfd7bcb2e191247640239cbc9c4546822113a142766502ade096R122

File: tests/jballerina-unit-test/src/test/resources/testng.xml [hunk 2]
```diff
@@ -252,11 +252,7 @@
                     <exclude name="testAnonStructAsStructField"/>
                 </methods>
             </class>
-            <class name="org.ballerinalang.test.object.ObjectTypeReferenceTest">
-                <methods>
-                    <exclude name="testInvalidTypeReferenceAcrossModules"/>
-                </methods>
-            </class>
+            <class name="org.ballerinalang.test.object.ObjectTypeReferenceTest"/>
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/resources/testng.xml:255]: I think these are already included in https://github.com/ballerina-platform/ballerina-lang/pull/26797/files#diff-3835b6074874bfd7bcb2e191247640239cbc9c4546822113a142766502ade096R122

File: tests/jballerina-unit-test/src/test/resources/testng.xml (hunk 1 above)
Comment (pramodya1994) [reply tests/jballerina-unit-test/src/test/resources/testng.xml]: Removed

File: tests/jballerina-unit-test/src/test/resources/testng.xml (hunk 2 above)
Comment (pramodya1994) [reply tests/jballerina-unit-test/src/test/resources/testng.xml]: Removed

**Review by hemikak — APPROVED** [review]

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectEquivalencyTest.java [hunk 3]
```diff
@@ -37,7 +37,7 @@
     public void setup() {
 
 //        compileResult = BCompileUtil.compile("test-src/object/ObjectProject", "ObjectEquivalencyTest");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectEquivalencyTest.java:39]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectTypeReferenceTest.java [hunk 4]
```diff
@@ -259,7 +259,7 @@ public void testTypeReferencedFunctionImplementation() {
     @Test
     public void testInvalidTypeReferenceAcrossModules() {
 //        CompileResult result = BCompileUtil.compile("test-src/object/ObjectProject", "object_reference_negative");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectTypeReferenceTest.java:261]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java [hunk 5]
```diff
@@ -35,31 +35,29 @@ public void testRuntimeObjEqNegative() {
 
 //        CompileResult compileResult = BCompileUtil.compile("test-src/object/ObjectProject",
 //                "object-private-fields-01-negative");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java:37]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java [hunk 6]
```diff
@@ -35,31 +35,29 @@ public void testRuntimeObjEqNegative() {
...
         Assert.assertEquals(returns[0].stringValue(), "{ballerina}TypeCastError {\"message\":\"incompatible types:" +
-                " 'org.foo:user' cannot be cast to 'object-private-fields-01-negative:userB'\"}");
+                " 'pkg.org_foo:user' cannot be cast to 'pkg:userB'\"}");
     }
 
     @Test(description = "Test private field access")
     public void testPrivateFieldAccess() {
 
 //        CompileResult compileResult = BCompileUtil.compile("test-src/object/ObjectProject",
 //                "object-private-fields-02-negative");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java:49]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java [hunk 7]
```diff
@@ -35,31 +35,29 @@ public void testRuntimeObjEqNegative() {
...
 
         BAssertUtil.validateError(compileResult, 0, "attempt to refer to non-accessible symbol 'ssn'", 7, 18);
-        BAssertUtil.validateError(compileResult, 1, "undefined field 'ssn' in object 'testorg/org.foo:1.0.0:person'",
+        BAssertUtil.validateError(compileResult, 1, "undefined field 'ssn' in object 'test/pkg.org_foo:1.0.0:person'",
                 7, 20);
     }
 
     @Test(description = "Test private object access in public functions")
     public void testPrivateObjAccess1() {
 //        CompileResult compileResult = BCompileUtil.compile("test-src/object/ObjectProject", "private-field1");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java:59]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java [hunk 8]
```diff
@@ -75,7 +73,8 @@ public void testPrivateObjAccess1() {
     @Test(description = "Test private object access in public functions")
     public void testPrivateObjAccess1SemanticsNegative() {
 //        CompileResult compileResult = BCompileUtil.compile("test-src/object/ObjectProject", "private-field1.sn");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java:75]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java [hunk 9]
```diff
@@ -94,20 +93,21 @@ public void testPrivateObjAccess1SemanticsNegative() {
...
         int i = 0;
         BAssertUtil.validateError(compileResult, i++, expectedErrMsg1 + "'address'", 10, 13);
         BAssertUtil.validateError(compileResult, i,
-                "undefined field 'address' in object 'testorg/org.foo.baz:1.0.0:FooEmployee'", 10, 18);
+                "undefined field 'address' in object 'test/pkg.org_foo_baz:1.0.0:FooEmployee'", 10, 18);
     }
 
     @Test(description = "Test private object access in public functions")
     public void testPrivateObjAccess2SemanticsNegative() {
 //        CompileResult compileResult = BCompileUtil.compile("test-src/object/ObjectProject", "private-field2.sn");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectWithPrivateFieldsNegativeTest.java:108]: Shall we remove this line ?

**Pull request merged**